src/boot/activity-detector/ActivityDetector.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { throttle } from 'lodash';
+import { connectIdmWallet } from 'react-idm-wallet';
+
+const THROTTLE_TIME = 10000;
```

Comment:
I think we can increase this number to something close to our locker minimum accepted time. This way, the restart will happen less often.

Reply:
Yep we can but this way we do not need to read the maxTime nor to react to changes to it.